testsuite/MDAnalysisTests/analysis/test_align.py

```diff
@@ -21,6 +21,7 @@
 # J. Comput. Chem. 32 (2011), 2319--2327, doi:10.1002/jcc.21787
 #
 from __future__ import absolute_import, division, print_function
+from contextlib import ExitStack as does_not_raise
```

Comment:
`ExitStack` requires Python `3.3`, but we *still* support Python `2.7` unfortunately. So, that is causing some CI issues.